programs/dasheri/src/pool/instructions/deposit_into_pool.rs

```diff
@@ -49,6 +49,9 @@ impl<'info> PoolDepositIntoPool<'info> {
     }
 }
 
+// REVIEW: What's this instruction good for? Why can I deposit into the vault?
```

Comment:
users would deposit into the pool, and a pool admin would deposit it into the pools mango account